magma/magma
# chore(nms): Upgrade material-UI to v5

## Summary

Upgrades material-UI and related dependencies. 
The default for inputs changed from underlined to outlined. This broke all places were we still use underlined inputs.
I have migrated a few places that felt very inconsistent to outlined inputs and fixed the remaining places.

There are many more breaking changes and the theme had to be reworked.
This is not a pixel perfect migration e.g. the spacing changed a bit.

- fixes #13270

## Test Plan

- On master open at least one tab per network type and one tab for the host organization in a secondary browser or private browsing mode.
- Checkout `nms/material-ui-5` and rebuild the docker container.
- Open the app in your primary browser and compare each page and dialog against master. There will be differences, they should make sense and it shouldn't look broken.
